packages/react-fela/docs/withTheme.md

```diff
@@ -25,3 +28,8 @@ const RenderTreeFragment = (
 
 // => <div>The primary color is red</div>
 ```
+## Note

```

Comment:
This could be renamed to Tips as we did that at some other places as well :)

Reply:
@rofrischmann changes done. Thanks for pointing them out 👍